### DISCUSSION TITLE
IDE suggestion

### DISCUSSION THREAD

**Discussion by D3vShip** [Cyfrin/foundry-full-course-cu#2286] [Q&A]
Hi, Just started learning [Solidity Smart Contract Development](https://updraft.cyfrin.io/courses/solidity). I need some suggestions regarding IDE here. 
Is Remix IDE, a browser-based complier and IDE is safe to use without any VPN ? if so, please confirm.
I'm just starting out in web3, so your guidance would be invaluable. 
Thanks!

**Comment by EngrPips** [comment]
Hello @D3vShip, I believe Remix IDE is a good one to use, and you won't be using Remix IDE in the later sections of the course. As Patrick will teach you how to develop locally on your machine using `VSCODE` you can choose to use any other code editor of your choice if you don't like `VSCODE`

**Reply by D3vShip** [reply]
Thank you for the prompt and insightful response!

**Reply by EngrPips** [reply]
You are welcome, Friend. I wish you the best on your journey.
